### ISSUE TITLE
Focus for accessibility

### ISSUE THREAD

**Issue by jllagerberg** [euvl/vue-js-toggle-button#127]
Is there a reason why there is no focus function for accessibility built in to the toggles?
I know this has been brought up before #26 #82.

I think this is really important and there isn't much work implementing it.

**Comment by jllagerberg** [comment]
Solved it for myself with three changes in the CSS:

```
.v-switch-input {
  opacity: 1 !important;
  width: 100% !important;
  height: 100% !important;
}
```

**Comment by jllagerberg** [comment]
PR: #128
